Header   header
uint16   chunk_size
uint8    chunk_number
float32  elapsed_time
bool     corrupted
string[] data
